### ISSUE TITLE
Use of Validation Set for Creation of the Memory Set

### ISSUE THREAD

**Issue by yashika-git** [ojedaf/vCLIMB_Benchmark#7]
Hello @ojedaf, 

Thanks for creating the vCLIMB Benchmark and for also providing the code. It's been really helpful.

I came across something in the code, specifically in the main_icarl_conLoss.py file, around line 214 ([here](https://github.com/ojedaf/vCLIMB_Benchmark/blob/05c14f4a990655cfb0df17788135c18b671adca9/main_icarl_conLoss.py#L214C15-L214C33)). I noticed that the validation set is being used to create the exemplar set (the memory).

I was curious about the choice of using the validation set here. Is there a particular reason for this, or should we consider using the training set to create the memory?

Thank you!

**Comment by ojedaf** [comment]
Dear @yashika-git, 

Thank you very much for your interest in our work. As seen in Line 214, the data (data parameter) used to fill the memory is from the training set for the current task (please see Line 201). We use val_cilDatasetList to create the dataloader using the validation data transformations to select the relevant videos to save into the memory.

Best,
AV
